package.json

```diff
@@ -23,11 +23,15 @@
     "docs": "typedoc",
     "docs:watch": "watch 'npm run docs' ./src",
     "lint": "eslint --ignore-path .gitignore '**/*.ts' --fix",
-    "server:samples": "http-server samples -c1 -p 3000 --ssl --cert certs/samples.crt --key certs/samples.key",
-    "server:e2e": "http-server tests/e2e/test-site -c1 -p 3000 --ssl --cert certs/samples.crt --key certs/samples.key",
+    "server:iframe": "http-server tests/e2e/iframe -c1 -p 3000 --ssl --cert certs/samples.crt --key certs/samples.key",
+    "server:samples": "http-server samples -c1 -p 3001 --ssl --cert certs/samples.crt --key certs/samples.key",
+    "server:e2e": "http-server tests/e2e/test-site -c1 -p 3002 --ssl --cert certs/samples.crt --key certs/samples.key",
+    "server:e2e:live": "http-server tests/e2e/test-site -c1 -p 3003 -e htm --ssl --cert certs/samples.crt --key certs/samples.key",
```

Comment:
Notice the `-e htm` option here. This runs the server with it pointing to the live AM version.